.jshintrc

```diff
@@ -1,7 +1,6 @@
 {
   "globalstrict": true,
   "bitwise": true,
-  "camelcase": true,
```

Comment:
This options is [deprecated](http://jshint.com/docs/options/#camelcase) and it's always complaining about some Janus code. Hopefully we could use [JSCS](http://jscs.info/) in the near future for styling issues.